src/sentry/static/sentry/app/components/subscribeButton.tsx

```diff
@@ -6,7 +6,14 @@ import Button from 'app/components/button';
 
 import {t} from 'app/locale';
 
-export default class SubscribeButton extends React.Component {
+type Props = {
+  onClick: (e: React.MouseEvent) => void;
+  disabled?: boolean;
+  isSubscribed?: boolean;
+  size?: 'zero' | 'micro' | 'small' | 'xsmall' | 'xxsmall' | 'large';
```

Comment:
These should come from the button props.

`Button['props']['size']`

Reply:
Can we just make this a union of `Button['props']` and then pass `rest` to the button

Reply:
Thanks, updated to use size from button. I don't want to change the implementation further right now.